httpev.ml

```diff
@@ -928,12 +928,14 @@ let send_reply c cout reply =
   end
   in
   (* possibly apply encoding *)
-  let (hdrs,body) =
+  let%lwt (hdrs,body) =
     (* TODO do not apply encoding to application/gzip *)
     (* TODO gzip + chunked? *)
     match body, code, c.req with
-    | `Body s, `Ok, Ready { encoding=Gzip; _ } when String.length s > 128 -> ("Content-Encoding", "gzip")::hdrs, `Body (Gzip_io.string s)
-    | _ -> hdrs, body
+    | `Body s, `Ok, Ready { encoding=Gzip; _ } when String.length s > 128 ->
```

Comment:
We allocate a 1024-byte buffer in `Gzip_stream`, so maybe a higher threshold makes sense (it feels a bit silly to go ahead and allocate 1K for only 129 bytes of response).